src/som/vmobjects/SInteger.java

```diff
@@ -24,19 +25,38 @@
 
 package som.vmobjects;
 
-import java.math.BigInteger;
-
 import som.vm.Universe;
 
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.Map;
+
 public final class SInteger extends SNumber {
 
+  /**
+   * Language convention requires integers up to this value to be identical.
+   */
+  private static final long MAX_IDENTICAL_INT = 1073741823L;
```

Comment:
Should we have tests for this property?
Looking briefly over https://github.com/SOM-st/SOM/blob/master/TestSuite/IntegerTest.som looks like we don't have it at the moment. If I didn't miss anything.

Note, if we got this as an optional requirement, we got `#expect:equals:` for example on [line 255](https://github.com/SOM-st/SOM/blob/master/TestSuite/IntegerTest.som#L255)